### ISSUE TITLE
send notifications to matrix

### ISSUE THREAD

**Issue by volodymyrss** [oda-hub/oda-bot#19]

**Comment by dsavchenko** [comment]
That's true in general, but I'm not sure for this particular case something useful would end up in matrix. 
This is a bug, see the last PR

**Comment by dsavchenko** [comment]
Duplicate of https://github.com/oda-hub/oda-bot/issues/15
